statistics/analyze_jobs.go

```diff
@@ -80,13 +80,13 @@ func ClearHistoryJobs() {
 // GetAllAnalyzeJobs gets all analyze jobs.
 func GetAllAnalyzeJobs() []*AnalyzeJob {
 	analyzeStatus.Lock()
+	defer analyzeStatus.Unlock()
```

Comment:
Thus, this change is not necessary.

Reply:
em...., i think it is also useful, because the jobs in `analyzeStatus` be appended in this func，which need a lock to prevent other operation change the `analyzeStatus`.

Reply:
But the sort operation need not in the critical section

Reply:
got it, addressed comment, @SunRunAway PTAL